File: src/class/Maridaje.js
export default class Maridaje {
  constructor(descripcion, nombre) {
    this.descripcion = descripcion;
    this.nombre = nombre;
  }

  // Métodos getter y setter para los atributos de la clase Maridaje
  get descripcion() {
    return this.descripcion;
  }

  set descripcion(descripcion) {
    this.descripcion = descripcion;
  }

  get nombre() {
    return this.nombre;
  }

  set nombre(nombre) {
    this.nombre = nombre;
  }
}

File: src/com/BonVino/proyect/Maridaje.java
package com.BonVino.proyect;

public class Maridaje {
    private String descripcion;
    private String nombre;

    // Constructor
    public Maridaje(String descripcion, String nombre) {
        this.descripcion = descripcion;
        this.nombre = nombre;
    }

    // Métodos getter y setter para descripcion
    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    // Métodos getter y setter para nombre
    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
}
